src/expressions/attribute.js

```diff
@@ -47,7 +47,7 @@ function canRenderAttribute(value) {
  * @returns {boolean} boolean - true if the attribute can be removed}
  */
 function shouldRemoveAttribute(value) {
-  return isNil(value) || value === false || value === ''
+  return !value && value !== 0
```

Comment:
Attributes with a NaN value will now be removed. I don't think it matters. I modified the test to check for all possible falsy values except 0.